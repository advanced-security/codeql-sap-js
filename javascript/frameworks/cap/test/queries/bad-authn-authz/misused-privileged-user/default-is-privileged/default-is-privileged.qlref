bad-authn-authz/DefaultUserIsPrivileged.ql