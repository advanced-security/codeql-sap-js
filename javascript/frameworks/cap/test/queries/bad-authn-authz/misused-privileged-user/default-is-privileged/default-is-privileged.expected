| server.js:8:1:8:36 | cdsUser ... vileged | The default user is being overridden to a privileged user. |
| srv/service1.js:14:7:14:42 | cdsUser ... vileged | The default user is being overridden to a privileged user. |
| srv/service2.js:12:5:12:40 | cdsUser ... vileged | The default user is being overridden to a privileged user. |
